frontend/src/metabase/query_builder/components/saved-question-picker/SavedQuestionPicker.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useMemo, useState, useCallback } from "react";
+import { Box } from "grid-styled";
+import _ from "underscore";
+import PropTypes from "prop-types";
+import { t } from "ttag";
+import { connect } from "react-redux";
+
+import Collection from "metabase/entities/collections";
+import Icon from "metabase/components/Icon";
+import { Tree } from "metabase/components/tree";
+
+import { SAVED_QUESTIONS_VIRTUAL_DB_ID } from "metabase/lib/constants";
+import Schemas from "metabase/entities/schemas";
+
+import SavedQuestionList from "./SavedQuestionList";
+import {
+  SavedQuestionPickerRoot,
+  CollectionsContainer,
+  BackButton,
+} from "./SavedQuestionPicker.styled";
+import { buildCollectionTree } from "./utils";
+
+const propTypes = {
+  onSelect: PropTypes.func.isRequired,
+  onBack: PropTypes.func.isRequired,
+  collections: PropTypes.array.isRequired,
+  schemas: PropTypes.array.isRequired,
+  databaseId: PropTypes.string,
+  tableId: PropTypes.string,
+};
+
+const OUR_ANALYTICS_COLLECTION = {
+  id: "root",
+  schemaName: "Everything else",
+  name: t`Our analytics`,
+  icon: "folder",
+};
```

Comment:
Should we maybe use the `ROOT_COLLECTION` object from [here](https://github.com/metabase/metabase/blob/ab50eaaa2fdf0583092f99ee11e8fc9a298582bf/frontend/src/metabase/entities/collections.js#L18), so we have a single source of truth on root ID and name?